src/client/components/forms/publication.js

```diff
@@ -47,21 +55,41 @@ class PublicationForm extends React.Component {
 		this.handleSubmit = this.handleSubmit.bind(this);
 	}
 
+	/**
+	 * Changes the current tab to the one specified and checks
+	 * if the alias form is valid.
```

Comment:
"refreshes the validity of the aliases tab in the form state" might give more context here